README.md

```diff
@@ -162,12 +162,12 @@ The code supports multiple mode of operation:
 
 * Training mode:
 ```ShellSession
-./build/src/phoebe -f ./csv_files/rates_trained_data.csv -m training -s settings.json
+./build/src/phoebe -f ./csv_files/rates.csv -m training -s settings.json
 ```
 
 * Inference
 ```ShellSession
-./build/src/phoebe -f ./csv_files/rates_trained_data.csv -i wlan0 -m inference -s settings.json
+./build/src/phoebe -f ./csv_files/rates.csv -i wlan0 -m inference -s settings.json
```

Comment:
Actually you want to use the trained data when running the inference...

Reply:
didn't know that this filename had some background meaning , for me this change was motivated by something more straightforward: after following README I copy/paste this line to check that everything is working but it failed because current master does not have `./csv_files/rates_trained_data.csv` but it has `./csv_files/rates.csv` :) 

ok I will revert this

Reply:
Don't revert! Make sure the TRAINING and INFERENCE sections use different file names.
Eventually the filenames could be called anything, it's up to the user, what matters is providing a meaningful example/suggestion on Phoebe usage.